pkg/crypto/eckey.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"fmt"
+	"math/big"
+)
+
+// ToECKey converts key to an ecdsa public key. It panics if the curve is invalid.
```

Comment:
Does this function really panic if the curve is invalid? It looks to me like it would only panic if `key` is `nil`.

Reply:
yeh we can't be panic ing!

Reply:
it's an outdated comment, it has been updated.